ui/src/shared/components/Notifications.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {PureComponent} from 'react'
+import {connect} from 'react-redux'
+import {Notification as NotificationType} from 'src/types/notifications'
+import Notification from 'src/shared/components/Notification'
+
+interface Props {
+  inPresentationMode?: boolean
+  notifications: NotificationType[]
+}
+
+class Notifications extends PureComponent<Props> {
+  public static defaultProps: Partial<Props> = {
+    inPresentationMode: false,
+  }
+
+  public render() {
+    const {notifications} = this.props
+
+    return (
+      <div className={this.className}>
+        {notifications.map(n => <Notification key={n.id} notification={n} />)}
+      </div>
+    )
+  }
+
+  private get className(): string {
+    const {inPresentationMode} = this.props
+
+    if (inPresentationMode) {
+      return 'notification-center__presentation-mode'
+    }
+
+    return 'notification-center'
+  }
+}
+
+const mapStateToProps = ({
+  notifications,
+  app: {
+    ephemeral: {inPresentationMode},
+  },
+}) => ({
```

Comment:
maybe we can add the return type of Props to this to enforce that it matches, typescript loses some type checking with redux and this would just add a additional check for us in the future.